internal/clients/team1/iito.go

```diff
@@ -0,0 +1,186 @@
+package team1
+
+import (
+	"math"
+	"sort"
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+)
+
+/**************************/
+/*** 		Types	 	***/
+/**************************/
+
+// optionOnTeamsDict contains the opinion of Team1 about other teams.
+// 0 is neutral, Positive -> Positive Opinion, Negative -> Negative Opinion
+type opinionOnTeams struct {
+	clientID shared.ClientID
+	opinion  int
+}
+
+type sortByOpinion []opinionOnTeams
+
+/**************************/
+/*** 		Helpers	 	***/
+/**************************/
```

Comment:
D: nitpick

Reply:
That's weird, it looks aligned on my computer.

Reply:
emacs reeeeee